tools/scripts/AksEdgeQuickStart/AksEdgeQuickStartForAio.ps1

```diff
@@ -49,18 +49,58 @@ function Restart-ApiServer
 {
 param(
     [Parameter(Mandatory=$true)]
-    [string] $serviceAccountIssuer
+    [string] $serviceAccountIssuer,
+    [Switch] $useK8s=$false
 )
 
     Write-Host "serviceAccountIssuer = $serviceAccountIssuer"
-    Invoke-AksEdgeNodeCommand -command "sudo cat /var/.eflow/config/k3s/k3s-config.yml | tee /home/aksedge-user/k3s-config.yml | tee /home/aksedge-user/k3s-config.yml.working > /dev/null"
-    Invoke-AksEdgeNodeCommand -command "sudo sed -i 's|service-account-issuer.*|service-account-issuer=$serviceAccountIssuer|' /home/aksedge-user/k3s-config.yml"
-    Invoke-AksEdgeNodeCommand -command "sudo cp /home/aksedge-user/k3s-config.yml /var/.eflow/config/k3s/k3s-config.yml"
-    Invoke-AksEdgeNodeCommand -command "sudo systemctl restart k3s.service"
+
+    if ($useK8s)
+    {
+        & kubectl delete pod -n kube-system -l component=kube-apiserver
+    }
+    else
+    {
+        Invoke-AksEdgeNodeCommand -command "sudo cat /var/.eflow/config/k3s/k3s-config.yml | tee /home/aksedge-user/k3s-config.yml | tee /home/aksedge-user/k3s-config.yml.working > /dev/null"
+        Invoke-AksEdgeNodeCommand -command "sudo sed -i 's|service-account-issuer.*|service-account-issuer=$serviceAccountIssuer|' /home/aksedge-user/k3s-config.yml"
+        Invoke-AksEdgeNodeCommand -command "sudo cp /home/aksedge-user/k3s-config.yml /var/.eflow/config/k3s/k3s-config.yml"
+        Invoke-AksEdgeNodeCommand -command "sudo systemctl restart k3s.service"
+    }
 
     Wait-ApiServerReady
 }
 
+function Verify-ConnectedStatus
+{
+param(
+    [Parameter(Mandatory=$true)]
+    [string] $resourceGroup,
+    [Parameter(Mandatory=$true)]
+    [string] $clusterName,
+    [Parameter(Mandatory=$true)]
+    [string] $subscriptionId
+)
+
+    $retries = 20
```

Comment:
should we try to make the total wait time = 10 mins? According to this doc, the total onboarding could be 5 to 10 mins. https://learn.microsoft.com/en-us/azure/azure-arc/kubernetes/quickstart-connect-cluster?tabs=azure-cli#verify-cluster-connection

Reply:
lets keep it same as Connect-AksEdgeArc. 600 secs.